### DISCUSSION TITLE
Setting up lessons admin

### DISCUSSION THREAD

**Discussion by storycollab** [ChurchApps/ChurchAppsSupport#371] [Lessons.church] [answered]
How do you set up the admin for lessons?

**Comment by storycollab** [comment]
Go to the gear in Chums and assign someone to this role; select the new role, assign people and then in that persons profile assign permissions.

**Comment by storycollab** [comment]
But still no admin in the drop-down in Lessons, as per the support docs- https://support.churchapps.org/lessons/admin/add-lesson.html; I was already the domain admin, but don't seem to have admin rights in lessons.

**Comment by jzongker** [comment, marked as answer]
The Lessons Admin role that gets created in Chums is to allow users to schedule lessons for their church on Lessons.church.

Setting up an actual lesson provider is very rare and there isn't a GUI for doing so yet.  For now, you have to manually add two database roles.
1. In membership.rolePermissions, create a new row with the churchId and roleId of the Lessons Admins group you just made.  Set the apiName to LessonsApi, contentType to Lessons and action to Edit.
2. In lessons.providers, create a new row for the churchId.  Set the display name and whether you're ready for the content to be live on the site.
